### PR TITLE
CSS Parser

### PR DESCRIPTION
**Pull request by zackradisic**
### What does this PR do?

you already know

### CODE REVIEW

**Comment by robobun** [comment]
@zackradisic, your commit fc6f8937c00cf4ab55a0cea408770f7fea5d64b3 is building: [`#3780`](https://buildkite.com/bun/bun/builds/3780)

File: src/bundler.zig [hunk 1]
```diff
@@ -1002,76 +1002,113 @@ pub const Bundler = struct {
             .dataurl, .base64 => {
                 Output.panic("TODO: dataurl, base64", .{}); // TODO
             },
+            // ZACK YOUR CODE GOES HERE!!
```
Comment (Jarred-Sumner) [inline src/bundler.zig]:
```suggestion

```

File: src/bun.js/javascript.zig [hunk 2]
```diff
@@ -2674,6 +2674,9 @@ pub const VirtualMachine = struct {
             )) {
                 .success => |r| r,
                 .failure => |e| {
+                    {
+                        @panic("LMAO fuck");
```
Comment (Jarred-Sumner) [inline src/bun.js/javascript.zig]: @zackradisic

Comment (zackradisic) [inline src/bun.js/javascript.zig:2678]:
```suggestion
```

Comment (zackradisic) [inline src/bun.js/javascript.zig]:
```suggestion

```

**Comment by mrbrianevans** [comment]
Out of curiosity, why was it necessary to port lightningcss to zig rather than calling the original through a foreign function interface?

**Pull request merged**